### PR TITLE
Runnable jar

### PR DESCRIPTION
**Pull request by tobiasschulz**

### CODE REVIEW

**Comment by dotModded** [comment]
This a large pr..

**Comment by tobiasschulz** [comment]
Not really just a lot of commits, and half of the changed lines is the patch file

File: nova.ps1 [hunk 1]
```diff
@@ -267,10 +266,44 @@ function Invoke-Nova {
...
+    git diff origin/minecraft-1.10-mcp > ..\..\..\..\patches\nova.patch 
 }
 
 ################################################################################
+#  install nova in minecraft launcher                                                             #
+################################################################################
+
+function install-Nova {
+    if (Test-Path  $Env:APPDATA\.minecraft\launcher_profiles.json){
+		new-item "$Env:APPDATA\.minecraft\versions\1.10-nova\" -ItemType Directory
```
Comment (DethRaid) [inline nova.ps1:278]: Didn't capitaliz the PowerShell commands :(

File: src/main/cpp/CMakeLists.txt [hunk 2]
```diff
@@ -168,6 +173,17 @@ endif()
 
 target_link_libraries(nova-renderer ${COMMON_LINK_LIBS})
 
+if (WIN32)
+add_custom_command(TARGET nova-renderer
```
Comment (DethRaid) [inline src/main/cpp/CMakeLists.txt:177]: nice!

File: src/main/cpp/render/nova_renderer.cpp [hunk 3]
```diff
@@ -84,8 +84,10 @@ namespace nova {
         // Render GUI objects
         std::vector<render_object *> gui_geometry = meshes.get_meshes_for_shader("gui");
         for(const auto *geom : gui_geometry) {
-            auto color_texture = textures.get_texture_atlas(geom->color_texture);
-            color_texture.bind(0);
+            if (geom->color_texture != texture_manager::texture_type::no_texture) {
```
Comment (DethRaid) [inline src/main/cpp/render/nova_renderer.cpp:87]: was hoping it would crash when the texture type was no_texture

**Review by DethRaid — APPROVED** [review]

**Pull request merged**